course_catalog/factories.py

```diff
@@ -125,31 +196,11 @@ class CourseRunFactory(DjangoModelFactory):
         )
     )
 
-    @factory.post_generation
-    def instructors(self, create, extracted, **kwargs):
-        """Create instructors for course"""
-        if not create:
-            return
-
-        if extracted:
-            for instructor in extracted:
-                self.instructors.add(instructor)
-
-    @factory.post_generation
-    def prices(self, create, extracted, **kwargs):
-        """Create prices for course"""
-        if not create:
-            return
-
-        if extracted:
-            for price in extracted:
-                self.prices.add(price)
-
     class Meta:
         model = CourseRun
 
 
-class BootcampFactory(DjangoModelFactory):
+class BootcampFactory(AbstractCourseFactory):
     """Factory for Bootcamps"""
 
     course_id = factory.Sequence(lambda n: "BOOTCAMP%03d.MIT" % n)
```

Comment:
The `BootcampSerializer` always sets `offered_by` to `OfferedBy.bootcamps.value`.  Maybe we should do the same for the factory?

```python
offered_by = OfferedBy.bootcamps.value
```